config/kafka-monitor.properties

```diff
@@ -46,7 +46,10 @@
     "topic": "kafka-monitor-topic",
     "zookeeper.connect": "localhost:2181",
     "bootstrap.servers": "localhost:9092",
+
     "produce.record.delay.ms": 100,
+    "produce.treat.zero.throughput.as.unavailable": false,
```

Comment:
Is that a good default? Any particular reason why we want this, if it wasn't the default in kafka-monitor to begin with?

Reply:
here is the description of that config : 
https://github.com/linkedin/kafka-monitor/blob/master/src/main/java/com/linkedin/kmf/services/configs/ProduceServiceConfig.java#L67

But yes I think, we should not introduce this here, we can add this in our run.py so that it will be signalfx specific.